### DISCUSSION TITLE
[QUESTION] Loading Xml file is throwing error

### DISCUSSION THREAD

**Discussion by RahulVKM96** [idea-statica/ideastatica-public#536] [Help Q&A] [answered]
I am Trying to Load an Xml file which is created by idea statica new version 24.0.1.1233. Here is the attached screen shot and attached xml file. Kindly help me to load this xml file.
![image](https://github.com/idea-statica/ideastatica-public/assets/105714132/f6ed3e08-1193-46c2-b5c8-f2eef204ae1b)

[trial.txt](https://github.com/idea-statica/ideastatica-public/files/15412518/trial.txt)
Since the .xml is not supported i am changing its extension as .txt and Uploading here. Please Look into this.

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @RahulVKM96, This file is a description of the class `OpenModelContainer`. The method you are using is specific for the class `OpenModel`. You can use `IdeaRS.OpenModel.Tools.OpenModelContainerFromFile(FinalXmlFile);`
